internal/iruntime/total_memory_linux.go

```diff
@@ -18,9 +18,13 @@ package iruntime
 
 import "go.opentelemetry.io/collector/internal/cgroups"
 
+// _unlimitMemorySize defines the bytes size when memory limit is not set
+// for the container and process with cgroups
+const _unlimitMemorySize = 9223372036854771712
```

Comment:
unlimited

Reply:
Why "_" as prefix?

Reply:
"_" is naming convention to indicate the variables for internal use. I guess I'm fine to remove it  since it is restricted by lowercase in go.